akka-sample-cluster-client-grpc-scala/src/main/scala/sample/cluster/client/grpc/ClusterClient.scala

```diff
@@ -1,83 +1,95 @@
 package sample.cluster.client.grpc
 
-import scala.concurrent.ExecutionContext
-import scala.concurrent.Future
-import scala.concurrent.Promise
-import scala.concurrent.duration._
-
 import akka.NotUsed
-import akka.actor.Actor
-import akka.actor.ActorLogging
-import akka.actor.ActorRef
-import akka.actor.ActorSystem
-import akka.actor.Props
-import akka.actor.Terminated
+import akka.actor.{
+  Actor,
+  ActorLogging,
+  ActorRef,
+  ActorSystem,
+  Props,
+  Terminated
+}
```

Comment:
Wat? 
Should we use a wilcard import instead?

Reply:
Not sure what my IntelliJ is smoking. I think a wildcard is bad in a sample though.

Reply:
Aha, it's after switching to community edition of IntelliJ. Reformatted with the Akka defaults now.